.github/CODEOWNERS

```diff
@@ -6,21 +6,21 @@
 ###########
 
 # Catch all for non-code project files
-* @rickwinter @joshfree
+* @rickwinter
 
 # SDK team
-/sdk/ @ahsonkhan @antkmsft @rickwinter @vhvb1989
+/sdk/ @rickwinter
```

Comment:
I reduced this down to minimize noise for the team.  If others would like extra noise you can add yourself back.